site/en/docs/crux/history-api/index.md

```diff
@@ -0,0 +1,617 @@
+
+## Data model
+
+This section details the structure of data in requests and responses.
+
+### Record
+
+A discrete piece of information about a page, or site. A record can have data that is specific for an identifier and for a specific combination of dimensions. A record can contain data for one or more metrics.
+
+### Identifiers
+
+Identifiers specify what records should be looked up. In CrUX these identifiers are webpages and websites.
+
+### Origin
+
+When the identifier is an origin all data present for all pages in that origin are aggregated together. For example, say the `http://www.example.com` origin had pages as laid out by this sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+This would mean that when querying the Chrome UX Report with the origin set to `http://www.example.com`, data for `http://www.example.com/`, `http://www.example.com/foo.html`, and `http://www.example.com/bar.html` would be returned, aggregated together, because those are all pages under that origin.
+
+### URLs
+
+When the identifier is a URL, only data for that specific URL will be returned. Looking again to the `http://www.example.com` origin sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+If the identifier is set to URL with the value of `http://www.example.com/foo.html`, only data for that page will be returned.
+
+### Dimensions
+
+Dimensions identify a specific group of data that a record is being aggregated against. For example, a form factor of `PHONE` indicates that the record contains information about loads that took place on a mobile device.
+
+The CrUX History API is only available aggregated by form factor dimension. This is a general class of device split into `PHONE`, `TABLET`, and `DESKTOP`.
+
+{% Aside %}
+Unlike the daily API, the CrUX History API does not have data aggregated against Effective Connection Type (ECT).
+{% endAside %}
+
+### Metric
+
+Metrics are expressed in a histogram array, which represents the percent of users that experienced a metric with that value proportionally to all.
+
+A simple three bin histogram for an example metric looks like this:
+
+```json
+{
+  "histogramTimeseries": [
+    {
+      "start": 0,
+      "end": 2500,
+      "densities": [0.9190, 0.9203, 0.9194, 0.9195, 0.9183, 0.9187]
+    },
+    {
+      "start": 2500,
+      "end": 4000,
+      "densities": [0.0521, 0.0513, 0.0518, 0.0518, 0.0526, 0.0527]
+    },
+    {
+      "start": 4000,
+      "densities": [0.0288, 0.0282, 0.0286, 0.0285, 0.0290, 0.0285]
+    }
+  ],
+}
+```
+
+This data indicates that 91.2% of users experience the example metric value between 0ms and 2,500ms for the first collection period in the history, followed by 92.0%, 91.9%, and so on. The units of the metric are not contained in this histogram, in this case we will assume milliseconds.
```

Comment:
Worth clarifying that "first collection period in the history" corresponds to the earliest period (index 0) in the `collectionPeriods` array.